Dockerfile

```
@@ -7,28 +7,28 @@
 FROM docker.mirror.hashicorp.services/alpine:latest AS release-default
 
 ARG BIN_NAME=consul-ecs
-ARG VERSION
+ARG PRODUCT_VERSION
 # TARGETARCH and TARGETOS are set automatically when --platform is provided.
 ARG TARGETOS TARGETARCH
 # Export BIN_NAME for the CMD below, it can't see ARGs directly.
 ENV BIN_NAME=$BIN_NAME
-ENV VERSION=$VERSION
+ENV VERSION=$PRODUCT_VERSION
 
 LABEL description="consul-ecs provides first-class integration between Consul and AWS ECS." \
       maintainer="Consul Team <consul@hashicorp.com>" \
       name=$BIN_NAME \
       release=$VERSION \
```

The value of `release` will get set to the same value as `$PRODUCT_VERSION` because it will grab the value from the `$VERSION` env var... but for consistency we should probably use `$PRODUCT_VERSION` directly instead of transitively.

```suggestion
      release=$PRODUCT_VERSION \
```